"Installing Wazuh with Elastic Stack" index update (#5882)
wazuh/wazuh-documentation: source/deployment-options/elastic-stack/index.rst

Installing Wazuh with Elastic Stack
===================================

This section aims to guide the user through the installation process of Wazuh. This installation guide will use the Elastic Stack basic license option, which contains everything included in the open-source version under the Apache 2.0 license, plus additional capabilities such as Elastic Stack Security features, Kibana alerting, and others.

> This guide includes instructions to install Wazuh |WAZUH_CURRENT| and Elastic Stack |ELASTICSEARCH_ELK_LATEST|.  For a complete list of compatible versions, check our :ref:`Compatibility matrix <compatibility_matrix_elk>` and :ref:`Packages list <packages_list_elk>`.  

Reviewer: ```suggestion
This guide includes instructions to install Wazuh |WAZUH_CURRENT| and Elastic Stack |ELASTICSEARCH_ELK_LATEST|.  For a list of compatible versions, check our :ref:`Compatibility matrix <compatibility_matrix_elk>` and :ref:`Packages list <packages_list_elk>`.  
```

| Kibana Version   | Package                  |
+==================+==========================+
| 7.16.0           | |WAZUH_KIBANA_7.16.0|    |
+------------------+--------------------------+
| 7.16.1           | |WAZUH_KIBANA_7.16.1|    |
+------------------+--------------------------+
| 7.16.2           | |WAZUH_KIBANA_7.16.2|    |
+------------------+--------------------------+
| 7.16.3           | |WAZUH_KIBANA_7.16.3|    |
+------------------+--------------------------+
| 7.17.0           | |WAZUH_KIBANA_7.17.0|    |
+------------------+--------------------------+
| 7.17.1           | |WAZUH_KIBANA_7.17.1|    |
+------------------+--------------------------+
| 7.17.2           | |WAZUH_KIBANA_7.17.2|    |
+------------------+--------------------------+
| 7.17.3           | |WAZUH_KIBANA_7.17.3|    |
+------------------+--------------------------+
| 7.17.4           | |WAZUH_KIBANA_7.17.4|    |
+------------------+--------------------------+
| 7.17.5           | |WAZUH_KIBANA_7.17.5|    |
+------------------+--------------------------+
| 7.17.6           | |WAZUH_KIBANA_7.17.6|    |
+------------------+--------------------------+

> For a complete list of the available versions, see the `Wazuh Kibana plugin compatibility matrix <https://github.com/wazuh/wazuh-kibana-app/wiki/Compatibility>`_.

Reviewer: ```suggestion
For versions older than |WAZUH_CURRENT|, check the full list in `Wazuh Kibana plugin compatibility matrix <https://github.com/wazuh/wazuh-kibana-app/wiki/Compatibility>`__.
```

| 7.10.2           | |WAZUH_KIBANA_7.10.2|    |
+------------------+--------------------------+
| 7.16.0           | |WAZUH_KIBANA_7.16.0|    |
+------------------+--------------------------+
| 7.16.1           | |WAZUH_KIBANA_7.16.1|    |
+------------------+--------------------------+
| 7.16.2           | |WAZUH_KIBANA_7.16.2|    |
+------------------+--------------------------+
| 7.16.3           | |WAZUH_KIBANA_7.16.3|    |
+------------------+--------------------------+
| 7.17.0           | |WAZUH_KIBANA_7.17.0|    |
+------------------+--------------------------+
| 7.17.1           | |WAZUH_KIBANA_7.17.1|    |
+------------------+--------------------------+
| 7.17.2           | |WAZUH_KIBANA_7.17.2|    |
+------------------+--------------------------+
| 7.17.3           | |WAZUH_KIBANA_7.17.3|    |
+------------------+--------------------------+
| 7.17.4           | |WAZUH_KIBANA_7.17.4|    |
+------------------+--------------------------+
| 7.17.5           | |WAZUH_KIBANA_7.17.5|    |
+------------------+--------------------------+
| 7.17.6           | |WAZUH_KIBANA_7.17.6|    |
+------------------+--------------------------+

> For a full list of the available Wazuh Kibana plugins, check the `Wazuh Kibana plugin compatibility matrix <https://github.com/wazuh/wazuh-kibana-app/wiki/Compatibility>`__.  

Reviewer: ```suggestion
For a full list of the available Wazuh Kibana plugin packages, check the `Wazuh Kibana plugin compatibility matrix <https://github.com/wazuh/wazuh-kibana-app/wiki/Compatibility>`__.  
```
